less/button.less

```diff
@@ -82,6 +82,7 @@ span.buttonSeparator {
     padding-bottom: 5px;
     width: auto;
     height: auto;
+    min-width: calc(@defaultFontSize * 6); // issue #6384
```

Comment:
Is it possible there are some buttons this will make unexpectedly bigger than what it was?